### PR TITLE
Script to Integrate RAETracker with TrackingMetrics

### PR DESCRIPTION
**Pull request by acoadmarmon**
Hey @ProfFan can you check out this PR to see if it's valid to merge? This script Andrew01.swift outputs the plots and images we went over on Friday. We will improve it over time, but I wanted to merge it since it does fulfill the functionality of integrating the RAETracker with TrackingMetrics to output metrics over n number of sequences.

### CODE REVIEW

**Review by ProfFan — APPROVED** [review]
Thanks Andrew and Brando!

**Comment by acoadmarmon** [comment]
Thanks for reviewing @ProfFan! Do you mind meeting this PR? I don't have write access to the repository so I can't merge it myself.

**Pull request merged**
